src/SIL.XForge.Scripture/ClientApp/src/xforge-common/json-api.service.ts

```diff
@@ -316,4 +317,11 @@ export class JSONAPIService {
       return ops;
     }, { update: [JSONAPIService.BACKUP] });
   }
+
+  changePassword(newPassword: string): Observable<any> {
```

Comment:
This should be implemented in `UserService`. Changing a user's password is already implemented in the JSON API backend, so you should be able to use the `updateAttributes` method from `JSONAPIService` to update the password. The method would look something like this:
``` ts
changePassword(newPassword: string): Promise<void> {
  return this.jsonApiService.updateAttributes(this.currentUserId, { password: newPassword }, false, true);
}
```
The last two boolean parameters in the `updateAttributes` call tells Orbit to not persist the change to IndexedDB and to block the promise until the server response comes back.